frontend/src/App.js

```diff
@@ -60,7 +63,11 @@ class App extends Component {
             console.log('LOGGED IN!')
             this.setState({token: localStorage.getItem('JWT')})
             axios.defaults.headers.common = { Authorization: `JWT ${localStorage.getItem('JWT')}`}
-            axios.get(CUR_USER).then(res => this.parseCurUser(res.data, false));
+            axios.get(CUR_USER,{
+                headers: {
+                    Authorization: `JWT ${localStorage.getItem('JWT')}`
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/StevenMonty/CS1530-Final/pullRequest?prid=7283612)
```suggestion
                    Authorization: `JWT ${localStorage.getItem("JWT")}`
```